demo/src/app/components/table/demos/sortable/table-sortable.ts

```diff
@@ -42,7 +42,8 @@ const COUNTRIES: Country[] = [
 export type SortColumn = keyof Country | '';
 export type SortDirection = 'asc' | 'desc' | '';
 const rotate: {[key: string]: SortDirection} = { 'asc': 'desc', 'desc': '', '': 'asc' };
-const compare = (v1: string, v2: string) => v1 < v2 ? -1 : v1 > v2 ? 1 : 0;
+
+const compare = (v1: string | number, v2: string | number) => v1 < v2 ? -1 : v1 > v2 ? 1 : 0;
```

Comment:
If I recall the shorter (and maybe more "idiomatic" way of writing this would be `compare = (v1: string | number, v2: string | number) => v1 - v2;`